packages/lambda-cog/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@basemaps/lambda-cog",
+  "version": "6.26.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/linz/basemaps.git",
+    "directory": "packages/lambda-cpg"
+  },
+  "author": {
+    "name": "Land Information New Zealand",
+    "url": "https://linz.govt.nz",
+    "organization": true
+  },
+  "type": "module",
+  "engines": {
+    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
+  },
+  "publishConfig": {
+    "access": "public"
```

Comment:
I dont think we need to publish this to NPM, can you set this package to "private: true" and remove the publish config

Reply:
Can we add this into package/server later?

Reply:
I dont think we need this to be published ever? well not as a lambda function anyway? this is all very specific to LINZ's internal storage structure.

Reply:
Done, thanks.